apps/todos/src/app/modules/about/containers/about/about.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'learning-workspace-about',
+  templateUrl: './about.component.html',
+  styleUrls: ['./about.component.scss'],
+})
+export class AboutComponent { }
```

Comment:
do we need about component?

Reply:
Yes it is, finally here will be information about this project